tests/test_roster.py

```diff
@@ -2,19 +2,21 @@
 
 def test_view_roster(client, auth):
     # getting the form as a logged in users,
-    # reguarless if the user owns the session or not
+    # reguarless if the user owns the roster or not
```

Comment:
getting the roster as a logged in teacher, students cant see it, but the auth.login() is a teacher in this case so it works. just maybe change the comment to logged in teacher